masterlist.yaml

```diff
@@ -159,6 +184,9 @@ common:
         text: '%1%が最新版ではないようです。一部のModを動作させるには最新版の%1%で追加された機能が必要となる場合があります。'
       - lang: de
         text: 'Sie haben anscheinend nicht die aktuellste Version von %1% installiert. Einige Mods könnten Funktionalitäten benötigen, die nur in der aktuellsten Version von %1% enthalten sind, um zu funktionieren.'
+      - lang: pt
+        text: 'Aparenta que você não tem a versão mais recente de %1%. Alguns mods podem requerir funcinalidades adicionadas na última versão de %1% para funcionar.'
+  
```

Comment:
The newline is fine, there's some extra white space, just remove the indent at the start of the line.


![image](https://user-images.githubusercontent.com/1944639/54870381-a716bc00-4d9d-11e9-8447-25122106690c.png)